accounts/abi/bind/precompiletemplate.go

```diff
@@ -0,0 +1,390 @@
+// (c) 2019-2022, Ava Labs, Inc. All rights reserved.
```

Comment:
nit: can we rename this file to `precompile_template.go` ?

Reply:
done